packages/core/src/tree.js

```diff
@@ -94,20 +95,29 @@ const Tree = props => {
         inputIconRenderer={inputIconRenderer}
         clearIconRenderer={clearIconRenderer}
       />
-      <Items styles={styles} getStyles={getStyles} height={itemsHeight}>
-        {leaves &&
-          leaves.length > 0 &&
-          leaves.map(item => (
-            <Item
-              getStyles={getStyles}
-              searchTerm={searchTerm}
-              item={item}
-              onClick={onClick}
-              forwardIconRenderer={forwardIconRenderer}
-              selectedItem={selectedItem}
-            />
-          ))}
-      </Items>
+      {customItemsRenderer ? (
+        React.cloneElement(customItemsRenderer, {
+          ...props,
```

Comment:
I'm not sure we should just pass the props (I'm talking about ...props), the props are meant for tree.
As any custom renderer, you can OTHER props externally when you do something like

```<Tree customItemsRenderer=<MyRenderer someProp={} />```

Unless you mean yo pass something like 
```
<Tree customItemsRenderer={someRenderer}/>
```
In which case you also need to create the component in the tree and you can still pass props using something like
```<Tree customItemsRenderer={() => <SomeRenderer .../>}/>
```

Hope my comment is understandable.